src/components/Navbar.tsx

```diff
@@ -2,9 +2,8 @@ import React, { useMemo, useState, useRef, useEffect } from 'react'
 import { useWeb3React } from '@web3-react/core'
 import { useTranslation } from 'react-i18next'
 import styled from 'styled-components'
-import { utils } from 'ethers'
 
-import { formatNumber, getTokenLogo, newTransactionsFirst, returnWalletAddress } from '~/utils'
+import { formatDataNumber, getTokenLogo, newTransactionsFirst, returnWalletAddress } from '~/utils'
```

Comment:
is formatNumber still being used in the app? Maybe worth removing if its no longer used to avoid confusion. I believe that was created by Iana, but maybe should have used `formatDataNumber` instead?